### ISSUE TITLE
Can't mount SMB share with space in name

### ISSUE THREAD

**Issue by nkm8** [pssc/ha-addon-lms#30]
I'm trying to mount an SMB share (I don't control the name of the share) that has a space in the name. When trying to mount the share, I get a "mount: bad usage" error. 

I've tried all iterations I can think of, but it looks like the quotes I add in the config aren't reflected when the mount command is executed.

**Comment by pssc** [comment]
You could try a custom start up command that runs a bash script with your
mount command

**Comment by nkm8** [comment]
Great suggestion - I tried this approach and everything is working great. 

It's up to you if you want to leave this issue open or not to improve the existing mount behavior. If you do, I suggest checking out how another developer handled a similar issue: https://github.com/alexbelgium/hassio-addons/blob/master/.templates/92-smb_mounts.sh

Thank you!

**Comment by pssc** [comment]
Yes it is something I need to fix and it is an issue, glad we found a work
round for you it allows me to pick this up as I have time.
